rotkehlchen/chain/ethereum/defi/zerionsdk.py

```diff
@@ -158,6 +158,56 @@ def _handle_pooltogether(normalized_balance: FVal, token_name: str) -> Optional[
     return None
 
 
+def _handle_pooltogether_v3(normalized_balance: FVal, token_name: str) -> Optional[DefiBalance]:
+    """Special handling for pooltogether v3"""
+    if 'COMP' in token_name:
+        comp_price = Inquirer.find_usd_price(A_COMP)
+        return DefiBalance(
+            token_address=to_checksum_address('0x27B85f596feB14e4B5fAA9671720a556a7608C69'),
```

Comment:
here and in the other places you would need to rebase on develop and use `string_to_ethereum_address` instead